packages/framework/src/Resources/config/services.yml

```diff
@@ -507,14 +507,6 @@ services:
             # must be run before ProductVisibilityFacade
             - { name: kernel.event_listener, event: kernel.response, method: onKernelResponse, priority: 20 }
 
-    Shopsys\FrameworkBundle\Model\Product\ProductFacade:
```

Comment:
I think this is still necessary

Reply:
I think Facade suffix is auto registered